### PR TITLE
New package: wqy-microhei-0.2.0_beta

### PR DESCRIPTION
**Pull request by rtlanceroad**

### CODE REVIEW

**Comment by xtraeme** [comment]
- The INSTALL script is unnecessary, your pkg is using **font_dirs** which uses the **x11-fonts** trigger.
- use vlicense

File: srcpkgs/wqy-microhei/template [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Template file for 'dejavu-fonts-ttf'
+pkgname=wqy-microhei
+version=0.2.0_beta
+revision=1
+short_desc="A Sans-Serif style high quality CJK outline font"
+maintainer="rtlanceroad <rtlanceroad@gmail.com>"
+homepage="http://wenq.org"
+license="GPL-3, APACHE"
```
Comment (xtraeme) [inline srcpkgs/wqy-microhei/template]: APACHE is not a correct a license, use **Apache-2.0**.

File: srcpkgs/wqy-microhei/template [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+distfiles="${SOURCEFORGE_SITE}/wqy/$pkgname/${version//_/-}/$pkgname-${version//_/-}.tar.gz"
+checksum=2802ac8023aa36a66ea6e7445854e3a078d377ffff42169341bd237871f7213e
+wrksrc=$pkgname
+
+depends="font-util fontconfig xbps-triggers>=0.58"
```
Comment (xtraeme) [inline srcpkgs/wqy-microhei/template]: This **depends** line is unnecessary, use simply `depends="font-util"`.

**Comment by rtlanceroad** [comment]
please test

File: srcpkgs/wqy-microhei/template [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+depends="font-util"
+noarch="yes"
+font_dirs="/usr/share/fonts/TTF"
+
+do_install() {
+	vmkdir usr/share/fonts/TTF
+	install -m644 *.ttc ${DESTDIR}/usr/share/fonts/TTF
+}
+post_install() {
+	vlicense LICENCE*
```
Comment (xtraeme) [inline srcpkgs/wqy-microhei/template]: This won't work, `vlicense` expects a single file. Also there's no reason to use `post_install`, move this to `do_install()`.

**Pull request merged**